TS3Hook/dllmain.cpp

```diff
@@ -204,6 +223,47 @@ void STD_DECL log_out_packet(char* packet, int length)
 
 		if (hConsole != nullptr) SetConsoleTextAttribute(hConsole, FOREGROUND_RED | FOREGROUND_BLUE | FOREGROUND_INTENSITY);
 	}
+	else if (find_pos_cinit != std::string::npos) 
+	{
+		const int client_ver = buffer.find("client_version=");
+		const int client_platform = buffer.find("client_platform=");
+		const int client_version_sign = buffer.find("client_version_sign=");
+		const int client_key_offset = buffer.find("client_key_offset=");
+		const int client_input_hardware = buffer.find("client_input_hardware=");
+		const int client_nickname = buffer.find("client_nickname=");
+		auto in_str = buffer;
+		if (clientver[2] != "") {
+			in_str.erase(client_version_sign + 20, (client_key_offset - client_version_sign - 21));
+			in_str.insert(client_version_sign + 20, clientver[2]);
+		}
+		if (clientver[1] != "") {
+			in_str.erase(client_platform + 16, (client_input_hardware - client_platform - 17));
+			in_str.insert(client_platform + 16, clientver[1]);
+		}
+		if (clientver[0] != "") {
+			in_str.erase(client_ver + 15, (client_platform - client_ver - 16));
+			in_str.insert(client_ver + 15, clientver[0]);
+		}
+		int nickname_length = (client_ver - client_nickname - 17);
+		
+		int length_difference = buffer.size() - in_str.size();
+		printf("LENGTH DIFFERENCE: %i\n", length_difference);
+		if (length_difference >= 0) {
+			memcpy(packet, in_str.c_str(), in_str.length());
+			memset(packet + in_str.length(), ' ', length - in_str.length());
+		}
+		else if (nickname_length > 3 && length_difference + nickname_length >= 0) {
+			in_str.erase(client_nickname + 16, (client_ver - client_nickname - 17));
+			in_str.insert(client_nickname + 16, "HAX");
```

Comment:
Please don't try to edit anything else than `client_version`, `client_platform` and `client_version_sign`

Reply:
This makes it possible because it gives it some space which it needs to go on some older versions eg 3.0.19.4

Reply:
hmm, can it be disabled?

Reply:
Done, check last commit